src/editors/handsontableEditor.js

```diff
@@ -4,59 +4,6 @@ import { setCaretPosition } from './../helpers/dom/element';
 import { stopImmediatePropagation, isImmediatePropagationStopped } from './../helpers/dom/event';
 import TextEditor from './textEditor';
 
-const HandsontableEditor = TextEditor.prototype.extend();
-
-/**
- * @private
- * @editor HandsontableEditor
- * @class HandsontableEditor
- * @dependencies TextEditor
- */
-HandsontableEditor.prototype.createElements = function(...args) {
-  TextEditor.prototype.createElements.apply(this, args);
-
-  const DIV = document.createElement('DIV');
-  DIV.className = 'handsontableEditor';
-  this.TEXTAREA_PARENT.appendChild(DIV);
-
-  this.htContainer = DIV;
-  this.assignHooks();
-};
-
-HandsontableEditor.prototype.prepare = function(td, row, col, prop, value, cellProperties, ...args) {
-  TextEditor.prototype.prepare.apply(this, [td, row, col, prop, value, cellProperties, ...args]);
-
-  const parent = this;
-  const options = {
-    startRows: 0,
-    startCols: 0,
-    minRows: 0,
-    minCols: 0,
-    className: 'listbox',
-    copyPaste: false,
-    autoColumnSize: false,
-    autoRowSize: false,
-    readOnly: true,
-    fillHandle: false,
-    autoWrapCol: false,
-    autoWrapRow: false,
-    afterOnCellMouseDown(_, coords) {
-      const sourceValue = this.getSourceData(coords.row, coords.col);
-
-      // if the value is undefined then it means we don't want to set the value
-      if (sourceValue !== void 0) {
-        parent.setValue(sourceValue);
-      }
-      parent.instance.destroyEditor();
-    }
-  };
-
-  if (this.cellProperties.handsontable) {
-    extend(options, cellProperties.handsontable);
-  }
-  this.htOptions = options;
-};
-
 const onBeforeKeyDown = function(event) {
```

Comment:
`onBeforeKeyDown` could be pulled into the editor class, similarly to [this example in the copyPaste plugin](https://github.com/handsontable/handsontable/blob/develop/src/plugins/copyPaste/copyPaste.js#L132).